recipe/meta.yaml

```
@@ -30,20 +36,23 @@ outputs:
         - setuptools
         - wheel
         - packaging
-        - param 1.9.0
+        - param =1.9
         - pyct 0.5.0
-        - bokeh 2.4.3
-        - nodejs 16.13.1
-        - pyviz_comms 0.7.2
-        - cartopy 0.21.1
+        - bokeh >=3.3.0,<3.4
+        - nodejs 18.16
+        - pyviz_comms >=0.6.0
       run:
         - python
-        - bokeh >=2.4.0,<2.5.0
+        - bokeh =3.3
```

```suggestion
        - bokeh >=3.2.0,<3.4.0
```
This doesn't match upstream.  Bokeh 3.2 is also allowed https://github.com/holoviz/geoviews/blob/6c889dd42501d11abea2377ce79990bf69806b99/setup.py#L97

---

Done in https://github.com/AnacondaRecipes/geoviews-feedstock/pull/11/commits/1b2879832057b2003c8e8a1e382195011a8463e6